README.md

```diff
@@ -9,6 +9,15 @@
 
 https://openebs.io/
 
+**Read this in**
+[🇩🇪](translations/README.de.md)	
+[🇷🇺](translations/README.ru.md)	
+[🇹🇷](translations/README.tr.md)	
+[🇺🇦](translations/README.ua.md)	
+[🇨🇳](translations/README.zh.md)	
+[🇫🇷](translations/README.fr.md)
+**[other languages](translations/#readme).**
```

Comment:
should the link be: https://github.com/openebs/openebs/blob/master/translations/TRANSLATIONS.md

Reply:
never mind. this is working as expected.